docs/content/intro/tutorial-preview-environments.md

```diff
@@ -0,0 +1,424 @@
+---
+title: "Tutorial: Preview Environments"
+description: Automate Creation of Development Preview Environments
+section: intro
+---
+
+In this tutorial we'll create a workflow for creating Development Preview Environments.
+
+Preview Environment is a dedicated Kubernetes Namespace where all of your applications 
+and their dependencies are deployed. We will create a pipeline for both creation of these ephemeral environments, 
+as well as auto-deployment of the applications themselves.
+
+Preview Environments are a great place for early experimentation where changes to your 
+application under development can be tested against production version of applications you're depending on.
+
+## Setup
+
+For the purpose of this tutorial we're assuming you have a Kubernetes cluster up and running and Brigade components are deployed to `brigade` namespace within it.  
+
+We'll be relying on GitHub webhooks so make sure that `brigade-github-app.enabled` is set to `true` when installing `brigade` helm chart. You can learn more about GitHub integration [here](../../topics/github)  
+
+[Docker for Desktop's Kubernetes](https://docs.docker.com/docker-for-mac/kubernetes/) 
+cluster is sufficient to perform all steps from this tutorial.  
+
+To accept incoming GitHub Webhooks you ensure your [ingress](../../topics/ingress/) is configured, alternatively, on desktop cluster you can use free version of [Ngrok](https://ngrok.com/) service to establish secure tunnelling. Follow this [excellent guide](https://stefanprodan.com/2018/expose-kubernetes-services-over-http-with-ngrok/) to set this up.
+
+Brigade's [brig](../../install/#brig) cli utility should be present on your machine.
+
+## Git Repositories
+
+We will use two GitHub repositories:
+
+- [brigade-tutorial-config](https://github.com/brigadecore/brigade-tutorial-config): containing orchestration responsible for managing Brigade projects and creating new environments.
+- [brigade-tutorial-app](https://github.com/brigadecore/brigade-tutorial-app): example microservice used to demonstrate release process.
+
+GitHub token will be used in this tutorial. To generate it follow [this article](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line).
+
+## Config Brigade Project
+
+Let's create new Brigade Project for our `brigade-tutorial-config` repository.
+
+```console
+$ brig project create --namespace brigade
+? Project Name brigadecore/brigade-tutorial-config
+? Full repository name github.com/brigadecore/brigade-tutorial-config
+? Clone URL https://github.com/brigadecore/brigade-tutorial-config
+```
+
+## Create Preview Environment Pipeline
+
+New environments will be created by executing brigade script via `brig` cli. 
+
+![Create Environment Pipeline](https://docs.brigade.sh/img/preview-environments-create.png)
```

Comment:
This image doesn't appear to be displaying...

![Screen Shot 2019-06-03 at 11 29 57 AM](https://user-images.githubusercontent.com/9322017/58821771-06414a00-85f3-11e9-95b7-960c53591f17.png)

Reply:
I wasn't sure if this was a limitation of the docs pipeline but I've followed the approach from how images are placed in existing doc e.g. https://github.com/brigadecore/brigade/commit/6d16368cdb3f4b49a7661ad9c398b13a63b6dbe1#diff-db7b9dcf2d24dfbc47fec1ae4e7ad0aaR106
Full image path seems to be used there.

Reply:
Ah, I see.  Indeed, this seems to be the way to do it. 👍